lib/parameters.py

```diff
@@ -343,3 +343,18 @@ def gen_vpp_rotation_parameters(spec):
   keys = ("case", "degrees")
   params = gen_vpp_rotation_variants(spec)
   return keys, params
+
+
+def gen_transcode_variants(spec, codec, mode):
+  for case, params in spec.iteritems():
+    variants = params.get("one_one", None)
```

Comment:
You can't iterate on `None` in the `for` loop below.

Reply:
added empty list for None scenario